experiments/compositions/composition/tests/scenario/scenario.go

```diff
@@ -153,11 +153,11 @@ func NewFromSample(t *testing.T, sample Sample, dependentSamples []Sample, hasCl
 }
 
 func (s *Scenario) Cleanup() {
+	defer cluster.ReleaseCluster(s.T, s.cluster)
+	defer s.GatherLogs()
```

Comment:
Assuming GatherLogs would be called first.

Reply:
Yes, first gather, then release.